Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -1638,10 +1638,17 @@ static bool isAuto(const LengthPoint& position)
     return position.x().isAuto() && position.y().isAuto();
 }
 
+static bool isNormal(const LengthPoint& position)
+{
+    return position.x().isNormal() && position.y().isNormal();
```

Comment:
I am puzzled why the `isAuto` function and this new function check both `x` and `y`. Clearly the code does not correctly support a mix of `normal` or `auto` and other types of length? Perhaps these should check one axis and assert the other. Or just check one axis.

Reply:
Changed to check x axis.